types/expressions.d.ts

```diff
@@ -2594,7 +2608,8 @@ declare module 'mongoose' {
   export type ConditionalExpressionOperator =
     Expression.Cond |
     Expression.IfNull |
-    Expression.Switch;
+    Expression.Switch |
+    Expression.Exists;
```

Comment:
No this is not a Conditional Expression Operator
see https://www.mongodb.com/docs/manual/reference/operator/aggregation/#conditional-expression-operators

Reply:
```suggestion
```

Reply:
I am not aware of all changes but it seems that `$exists` is not accepted anymore by the typings and despite the fact that it is not shown in the MongoDB documentation (oversight?) I think it is still a valid operator for `$match` query in the aggregation pipeline.

Wherever you put it, I think this should work:
```ts
const result = await Model.aggregate([{
  $match: { optional: { $exists: true } }
}]);
```
mongoose@6.4.0:
```
Type '{ $exists: true; }' is not assignable to type 'Expression'.
```